src/components/network/access_point.py

```diff
@@ -0,0 +1,13 @@
+import network
+
+# Creates a simple wifi access point to eventually use with sockets
+class AccessPoint():
```

Comment:
at some point we will want the AP name to be something detectable and parsable (lamp:encodedstring) where encoded string can be decoded to fetch name and colors

that way we can build a longer range network of lamps in an area outside the very short BT network, and also we can eventually use this for larger messaging as well as updates etc